packages/babel-preset-babili/__tests__/preset-tests.js

```diff
@@ -0,0 +1,42 @@
+jest.autoMockOff();
+
+const babel = require("babel-core");
+const unpad = require("../../../utils/unpad");
+
+function transform(code, options = {}, sourceType = "script") {
```

Comment:
Oh right, I forgot :) If you can make a PR for that, I'd gladly merge — would make tests a bit cleaner.